### PR TITLE
feat(fe-fpm-writer): Custom action. Empty string 'anchor' property created if position provided without 'anchor'.

### PR DESCRIPTION
**Pull request by 815are**
Bug fix for issue, when we pass `position` without `anchor` in `setting` object to `generateCustomAction` as:
```
 settings: {
       ...settings,
       position: {
            placement: 'Before'
       }
}
```

problem - in manifest it is created as:
```
"position": {
      "placement": "Before",
      "anchor": ""
},
```

correct way should be:
```
"position": {
      "placement": "Before"
},
```

### CODE REVIEW

File: .changeset/witty-boats-divide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': minor
```
Comment (zdravko-georgiev) [inline .changeset/witty-boats-divide.md:2]: Do we really need minor version bump? Isn't `patch` sufficient?

**Review by zdravko-georgiev — APPROVED** [review]
Code looks good, just one comment regarding the version bump

**Review by stefanschreck — APPROVED** [review]

File: .changeset/witty-boats-divide.md (hunk 1 above)
Comment (stefanschreck) [reply .changeset/witty-boats-divide.md:2]: that's the question indeed

Comment (815are) [reply .changeset/witty-boats-divide.md]: thanks, makes sense, I was not aware of `patch`.
Corrected to `patch` with commit 26c545bef8dbbf0a7449d80f247b4f092aa1fec7

**Review by stefanschreck — APPROVED** [review]

**Review by zdravko-georgiev — APPROVED** [review]

**Pull request merged**